### DISCUSSION TITLE
How to find and use WebP library?

### DISCUSSION THREAD

**Discussion by ghost** [JulNadeauCA/BSDBuild#16] [General]
https://packages.msys2.org/package/mingw-w64-x86_64-libwebp

**Comment by JulNadeauCA** [comment]
A new test module `webp` was added in https://github.com/JulNadeauCA/BSDBuild/commit/f23e49abccd3893c1a215f0f5f39a9c6557beb75. It tests for `libwebp` as well as `libsharpyuv`, `libwebpdemux` and `libwebpmux`. It has been tested with webp 1.3.2.
Thanks!
